etc/profile-m-z/tvnamer.profile

```diff
@@ -0,0 +1,72 @@
+# Firejail profile for tvnamer
+# Description: Automatic TV episode file renamer
+quiet
+# Persistent local customizations
+include tvnamer.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.config/tvnamer
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-proc.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/tvnamer
+mkfile ${HOME}/.config/tvnamer/tvnamer.json
```

Comment:
```suggestion
```

Considering the `mkdir`, is this needed?

Reply:
Nice catch, it's indeed superfluous. Removed!